app-api/src/main/resources/liquibase-sender.xml

```diff
@@ -351,4 +351,17 @@
         <addNotNullConstraint tableName="sender_sync_request" columnName="status" columnDataType="VARCHAR(50)" />
     </changeSet>
     
+    <changeSet author="dchirinda" id="20220929-1500">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists columnName="event_date" tableName="sender_sync_message" />
+            </not>
+        </preConditions>
+        <comment>Add event_date column to sender_sync_message table</comment>
+
+        <addColumn tableName="sender_sync_message">
```

Comment:
We should mark this as required

Reply:
Since this table is not in production, you should then add this column directly on the original changeset that creates the table